extensions/resource-deployment/src/test/index.ts

```diff
@@ -9,7 +9,6 @@ const testRunner = require('vscodetestcover');
 const suite = 'resource-deployment Extension Tests';
 
 const mochaOptions: any = {
-	ui: 'tdd',
```

Comment:
From @Charles-Gagnon on https://github.com/microsoft/azuredatastudio/pull/13183
Let's keep this specific and put 'bdd' for the ones we want to use bdd.

Reply:
I wanted to use bdd for all of them. I already changed all the existing ones to bdd also. In the future, we can put 'tdd' on the ones that we desire to use 'tdd'

Reply:
changed the value to ui: 'bdd' instead of removing to make it default to 'bdd'